### PR TITLE
Redefining Security Schemes in service store

### PR DESCRIPTION
**Pull request by jinanisha**
#### Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: legend-engine-xt-serviceStore-executionPlan/src/main/java/org/finos/legend/engine/plan/execution/stores/service/ServiceExecutor.java [hunk 1]
```diff
@@ -110,40 +110,39 @@ public static InputStreamResult executeHttpService(String url, List<ServiceParam
...
+        headers.forEach(requestBuilder::addHeader);
 
         try
         {
             HttpClientBuilder clientBuilder = HttpClients.custom();
 
             if (securitySchemes != null)
             {
-                securitySchemes.forEach(securityScheme -> processSecurityScheme(clientBuilder, profiles, securityScheme));
+                securitySchemes.forEach(securityScheme -> processSecurityScheme(clientBuilder, requestBuilder, profiles, securityScheme,authSpecs.get(securityScheme.id)));
```
[the remark this answers is not in the document]
Comment (epsstan) [reply legend-engine-xt-serviceStore-executionPlan/src/main/java/org/finos/legend/engine/plan/execution/stores/service/ServiceExecutor.java:141]: We have to document somewhere that we do (or do not) care about the order of the security schemes

File: legend-engine-xt-serviceStore-pure/src/main/resources/core_servicestore/executionPlan/executionPlan_print.pure [hunk 2]
```diff
@@ -34,7 +34,7 @@ function meta::external::store::service::executionPlan::toString::serviceStoreEx
    $space+'  mimeType = '+$s.mimeType +'\n'+
    $space+'  params = '+$s.params->map(p | $p->serviceParamToString())->joinStrings('[', ', ' ,']') +'\n'+
    if($s.requestBodyDescription->isNotEmpty(), |$space+'  requestBodyDescription = ' + $s.requestBodyDescription->toOne().resultKey + '(' + $s.requestBodyDescription->toOne().mimeType +')\n', |'') +
-   $space+'  securitySchemes = '+$s.securitySchemes->map(sec | $sec->securitySchemeToString($extensions))->joinStrings('[', ', ' ,']') +'\n'+
+   //$space+'  securitySchemes = '+$s.securitySchemes->map(sec | $sec->securitySchemeToString($extensions))->joinStrings('[', ', ' ,']') +'\n'+
```
Comment (epsstan) [inline legend-engine-xt-serviceStore-pure/src/main/resources/core_servicestore/executionPlan/executionPlan_print.pure:37]: remove comment

File: legend-engine-xt-serviceStore-pure/src/main/resources/core_servicestore/protocols/pure/vX_X_X/extension/extension_serviceStore.pure [hunk 3]
```diff
@@ -31,10 +32,10 @@ function meta::protocols::pure::vX_X_X::extension::store::service::getServiceSto
 {
    ^meta::protocols::pure::vX_X_X::extension::SerializerExtension_vX_X_X
    (
-      transfers_store_transformConnection = 
+      transfers_store_transformConnection2 = { extensions:meta::pure::extension::Extension[*] |
```
Comment (epsstan) [inline legend-engine-xt-serviceStore-pure/src/main/resources/core_servicestore/protocols/pure/vX_X_X/extension/extension_serviceStore.pure:35]: variable name

**Review by epsstan — CHANGES_REQUESTED** [review]

**Comment by finos-admin** [comment]
This PR is stale because it has been open for 30 days with no activity. Please remove stale label or add any comment to keep this open. Otherwise this will be closed in 5 days.

**Comment by finos-admin** [comment]
This PR was closed because it has been inactive for 35 days. Please re-open if this PR is still relevant.

**Pull request closed without merging**